src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs

```diff
@@ -16,8 +16,8 @@ public class DocumentationBrowserViewModel : NotificationObject, IDisposable
         #region Constants
 
         private const string HTML_TEMPLATE_IDENTIFIER = "%TEMPLATE%";
-        private const string BUILT_IN_CONTENT_INTERNAL_ERROR_FILENAME = nameof(Resources.InternalError) + ".html";
-        private const string BUILT_IN_CONTENT_FILE_NOT_FOUND_FILENAME = nameof(Resources.FileNotFound) + ".html";
+        private const string BUILT_IN_CONTENT_INTERNAL_ERROR_FILENAME = "InternalError.html";
+        private const string BUILT_IN_CONTENT_FILE_NOT_FOUND_FILENAME = "FileNotFound.html";
```

Comment:
is there another one of these somewhere for NoContent?

Reply:
oh right below this...

Reply:
The fact those files were included as resources was unnecessary. They were also embedded resources, so that was redundant. Too many hands on this feature was probably the cause :)